app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -1157,7 +1168,7 @@ export class Dispatcher {
 
       this.statsStore.recordRebaseSuccessAfterConflicts()
 
-      await this.completeRebase(
+      this.completeRebase(
```

Comment:
Shouldn't this be returned or awaited?

Checking where this method is called, it seems to end up in the `ShowProgressStep`'s `rebaseAction` property, which returns a promise.

Am I missing something? 🤔

Reply:
Weirdness.. It looks like I somehow cut the return statement and pasted it below..